src/components/DropColumn.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Character from './Character.jsx';
+import { Droppable } from 'react-beautiful-dnd';
+
+// Single column of all the unassigned Characters, is droppable so characters can be returned to it
+const DropColumn = ({ column, characterList }) => {
+  const style = {
+    height: 'fit-content'
+  };
+  if (column.id === 'unassignedCharacters') {
+    style.overflow = 'visible';
+    style.width = '220px';
+  }
```

Comment:
This would be _much_ better handled by a css class applied from outside this component, rather than the component peeking at the data it displays to alter its behavior. 
Ex:
```jsx
return <DropColumn key={column.id} column={column} characterList={characterList} className="actorDropColumn"/>;
and
<DropColumn key={columns['unassignedCharacters'].id} ... className="unassignedDropColumn />
```
with
```css
.unassignedDropColumn div {
    overflow: visible;
    width: 220px;
}
```
Peeking at data to alter behavior of a function at a different abstraction level can make code more complicated to maintain.   

Imagine we had a `sort()` function that took an array and returned it sorted.  
This `sort` function always behaves the same, it will sort elements by their natural ordering given any array. The behavior is easy to understand. Pass array, it sorts it.    
  
If we needed to add some functionality to our program that puts a title at the beginning of our sorted list, we could do it two ways.  
We could modify our `sort` function to always put the title element first or add the title element to the beginning of the list after it was sorted.  
When we modify the `sort` function in that way (adding program specific functionality to a generic function), we break our abstraction.   
  
If we use `sort` elsewhere, we need to remember to account for the title specific behavior.  
Does this page have a title?  
Does this page have a title, but at the bottom this time?  
Does this page have a element that happens to be a title, and it needs to be sorted normally and not put to the beginning?  
  
  
Matching code to the level of abstraction existing code is working at helps to make a project easier to understand and maintain. As another (tangential) example:
When api abstraction level differs:
```js
function enableAbc() {
    db.insert('enabled', 'abc');
}
function remove(key, value) {
    db.remove(key, value);
}
function getAbcValue() {
    return 'abc';
}
```
```js
// enable abc
enableAbc(); 
// disable abc
remove('enabled', getAbcValue());
```
vs when api abstraction level matches:
```js
function enableAbc() {
    db.insert('enabled', getAbcValue());
}
function disableAbc() {
    db.remove('enabled', getAbcValue());
}

function getAbcValue() {
    return 'abc';
}
```
```js
// enable abc
enableAbc(); 
// disable abc
disableAbc();
```

Reply:
Stellar example, and a great reminder to be striving for more modular code and pure functions.